docs/01-Chart-Configuration.md

```diff
@@ -257,6 +257,18 @@ afterBody | `Array[tooltipItem], data` | Text to render after the body section
 beforeFooter | `Array[tooltipItem], data` | Text to render before the footer section
 footer | `Array[tooltipItem], data` | Text to render as the footer
 afterFooter | `Array[tooltipItem], data` | Text to render after the footer section
+dataPoints | `Array[CustomTooltipDataPoint], data` | List of matching point informations.
+
+#### CustomTooltipDataPoint
```

Comment:
Ah, ok. That's my bad, I should have been clearer. If you can merge the doc sections and fix the merge conflict I'll merge this